sherpa-onnx/csrc/online-transducer-model-config.h

```diff
@@ -12,15 +12,25 @@ namespace sherpa_onnx {
 
 struct OnlineTransducerModelConfig {
   std::string encoder;
+  const char *encoder_buf_begin, *encoder_buf_end;
```

Comment:
```suggestion
  const char *encoder_buf_begin = nullptr;
  const char *encoder_buf_end = nullptr;
```

1. Please follow our existing code style to define one variable per line.
2. Please initialize it to `nullptr`.